### PR TITLE
Add test coverage

### PR DESCRIPTION
**Pull request by daniel-thom**

### CODE REVIEW

File: src/base.jl [hunk 1]
```diff
@@ -944,10 +944,6 @@ function make_time_series(
 end
 =#
 
-function get_time_series_names(::Type{T}, component::Component) where {T <: TimeSeriesData}
```
Comment (jd-lara) [inline src/base.jl:947]: This is a useful one to restore in the future.

Comment (daniel-thom) [reply src/base.jl:947]: It still exists in IS. If we want it then we need to re-export it in PSY.

Comment (jd-lara) [reply src/base.jl:947]: Yes, please.

**Comment by jd-lara** [comment]
@daniel-thom great, this PR restores the Coverage levels.

**Review by jd-lara — APPROVED** [review]

**Comment by jd-lara** [comment]
@daniel-thom do you mind removing nightly from https://github.com/NREL-SIIP/PowerSystems.jl/blob/master/.github/workflows/pr_testing.yml also?

**Comment by daniel-thom** [comment]
> 
> 
> @daniel-thom great, this PR restores the Coverage levels.

I'm not sure about restoration. This code was not tested previously. I don't know what caused the coverage drop.

**Pull request merged**